react-blog/src/services/posts.ts

```diff
@@ -1,9 +1,9 @@
-import { api } from "./configs/http";
+import axios from "axios";
 
 class PostsService {
   getAll() {
-    return api.get("/blog/posts/");
+    return axios.get("https://studapi.teachmeskills.by/blog/posts/?limit=100");
```

Comment:
Our API have base url: `https://studapi.teachmeskills.by/`
Also u can set params for your query at the second arg:
```
axios.get("https://studapi.teachmeskills.by/blog/posts/", {
params: { limit: 100}
});
```